### PR TITLE
Consistent capitalization

### PR DESCRIPTION
**Pull request by pinamiranda**
Solve https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/issues/177 issue.

All "Section" and "Article" are capitalised.

Added other capitalisations for consistency (referenced in https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/pull/134) - see the description of the commits.

### CODE REVIEW

File: docs/annexes/annex-2/annex-2-high-level-requirements.md [hunk 1]
```diff
@@ -1210,7 +1210,7 @@ B.  Restore Requirements 
 
 The most relevant scenario for this functionality is when a user would
 like to migrate from one Wallet solution to a different Wallet solution.
-Another scenario would be when the Wallet solution provider ceases to
+Another scenario would be when the Wallet solution Provider ceases to
```
Comment (skounis) [inline docs/annexes/annex-2/annex-2-high-level-requirements.md:1213]: @pinamiranda Why should "provider" be capitalized in a case like this? In this text flow, "provider" is a common noun, part of the sentence, and does not refer to any specific or special provider. 

Typically, we capitalize words when they are not part of the natural sentence flow, and we need to emphasize that they refer to something previously defined elsewhere. 

### Examples on capitalization:

- **Common Noun**: When the noun is not specific and is used in a general sense.
  - "... the details are defined in the relevant article." (article is not specific)

- **Proper Noun**: When the noun refers to a specific entity previously defined.
  - "... more details about this topic are provided in **Article** 5.1." (Article 5.1 is a specific section)

### Application to our Case

In our context, "provider" would typically be a common noun and not capitalized unless it refers to a specific entity previously defined or agreed upon to be treated as a proper noun within the document or context. Here are some examples of when "provider" might be capitalized:

1. **Common Noun**:
   - "The **provider** will be responsible for delivering the services." (general reference to any provider)

2. **Proper Noun**:
   - "The **Provider** has specific responsibilities outlined in Section 4." (if "Provider" is defined as a specific entity in a contract or legal document)

File: docs/annexes/annex-2/annex-2-high-level-requirements.md [hunk 2]
```diff
@@ -1357,7 +1357,7 @@ the public sector.  
...
+| QTSPAS_07 | The Commission SHALL publish, in agreement with the European Digital Identity Cooperation Group, a technical specification, referring to applicable standards, specifications and procedures, that will cover at least the attributes specified in Annex VI, wherever those attributes rely on authentic sources within the public sector, for which Member States should ensure that measures are taken to allow qualified Providers of electronic attestations of attributes to verify by electronic means, at the request of the user, their authenticity against the relevant authentic source. <p><br> Note: These standards and procedures should be, whenever possible, aligned and compatible with those used for the platforms implementing the Once Only Principle (OOTS). There is a clear synergy of both data exchange approaches. In fact, the national OOP node would be a candidate for acting as an intermediary between QTSPs issuing QEEAs and the Authentic Sources. |
```
Comment (skounis) [inline docs/annexes/annex-2/annex-2-high-level-requirements.md]: if we need to use a capitalization strategy here I would expect "Qualified Providers", "Electronic Attestations of Attributes" etc. and not just the word "Providers"

**Review by skounis — CHANGES_REQUESTED** [review]
@pinamiranda I think we should consider again the strategy on how we choose the words that will be capitalized 

cc @paolo-de-rosa @duviner @digeorgi

**Review by skounis — CHANGES_REQUESTED** [review]
@pinamiranda It's important to focus on British rather than American spelling. We require British spelling.

cc @paolo-de-rosa @duviner @digeorgi

**Review by skounis — APPROVED** [review]

File: docs/index.md [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 Under the [Electronic Identification, Authentication and Trust Services (eIDAS)
 Regulation](https://digital-strategy.ec.europa.eu/en/policies/eidas-regulation),
-EU Member States may, on a voluntary basis, notify and recognise, national
+EU Member States may, on a voluntary basis, notify and recognize, national
```
[the remark this answers is not in the document]
Comment (digeorgi) [reply docs/index.md:11]: Updated in accordance, directly in the branch

File: docs/annexes/annex-3/annex-3.01-pid-rulebook.md [hunk 4]
```diff
@@ -379,7 +379,7 @@ specified in ISO/IEC 18013-5:2021 \[ISO18013-5\] and the format
 specified in \[SD-JWT VC\] must be used. For the latter, attributes must
 be encoded in JSON. For the former, attributes must be encoded in CBOR.
 
-This section therefore specifies two separate encodings for the PID data
+This Section therefore specifies two separate encodings for the PID data
```
Comment (skounis) [inline docs/annexes/annex-3/annex-3.01-pid-rulebook.md:382]: II don't think we need to capitalize this.

File: docs/arf.md [hunk 5]
```diff
@@ -293,7 +293,7 @@ journey in this dynamic sector.
 This chapter describes the EUDI Wallet ecosystem as it
 is foreseen in the Commission's legislative proposal. The different
 roles in the EUDI Wallet ecosystem are described in figure 1 and
-detailed in the following sections.
+detailed in the following Sections.
```
Comment (skounis) [inline docs/arf.md:296]: I do not think we should capitalize this.

File: docs/arf.md [hunk 6]
```diff
@@ -1041,26 +1041,26 @@ as follows:
...
     attestations of attributes on behalf of the public sector bodies
-    responsible for authentic sources in accordance with Article 45f and
+    responsible for Authentic Sources in accordance with Article 45f and
     with Annex VII of the Regulation.
 
 -   **Non-Qualified EAA:** An EAA which is not QEAA or PuB-EAA.
 
 Please note that the differences between these types of attestation are
 purely legal. For example, a diploma may be a QEAA or a non-qualified
 EAA, depending on whether it is issued by a qualified trust service
```
Comment (skounis) [inline docs/arf.md:1056]: It looks like we should also capitalize the "Qualified Trust Service"

File: docs/arf.md [hunk 7]
```diff
@@ -1041,26 +1041,26 @@ as follows:
...
-provider (QTSP) or by an unqualified one. Similarly, an mDL may be
+Provider (QTSP) or by an unqualified one. Similarly, an mDL may be
 issued as a PuB-EAA, a QEAA, or a non-qualified EAA, depending on the
 legal status of the party issuing mobile driving licenses in each Member
-State, in addition to being an authorised mDL Provider in accordance
+State, in addition to being an authorized mDL Provider in accordance
```
Comment (skounis) [inline docs/arf.md:1060]: We should keep the British spelling `authorised`

**Review by skounis — CHANGES_REQUESTED** [review]
Some words should not be capitalized.
We need to keep the British spelling with `s` instead of `z`

**Review by skounis — CHANGES_REQUESTED** [review]
This PR also introduces the same changes as the following one, which has already been merged:
* https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/pull/213/files

To prevent conflicts, we should rebase this branch and apply the changes on top of the updated `main` branch.

File: docs/arf.md (hunk 5 above)
Comment (digeorgi) [reply docs/arf.md:296]: Agreed, updated in accordance.

File: docs/arf.md (hunk 7 above)
Comment (digeorgi) [reply docs/arf.md:1060]: Updated in accordance

File: docs/arf.md [hunk 8]
```diff
@@ -1041,26 +1041,26 @@ as follows:
...
 with the rules applicable for mDL Providers.
 
-### 5.2 Available standardised formats
+### 5.2 Available standardized formats
```
[the remark this answers is not in the document]
Comment (digeorgi) [reply docs/arf.md:1063]: Updated in accordance

File: docs/arf.md [hunk 9]
```diff
@@ -1653,7 +1653,7 @@ relationships are established:
     Wallet Solution, the User would not be able to use the Wallet
     Instance for obtaining those PID(s) or attestation(s).
 
-The next two sections discuss these trust relationships.
+The next two Sections discuss these trust relationships.
```
[the remark this answers is not in the document]
Comment (digeorgi) [reply docs/arf.md:1656]: it has been updated in accordance

File: docs/arf.md [hunk 10]
```diff
@@ -2591,7 +2591,7 @@ certification) against requirements which will be established by:
     down in [Article 5a](https://eur-lex.europa.eu/legal-content/EN/ALL/?uri=CELEX:32024R1183#d1e1347-1-1)(4), (5), (8);
 
 -   [Art. 5a](https://eur-lex.europa.eu/legal-content/EN/ALL/?uri=CELEX:32024R1183#d1e1347-1-1)(14) - Logical separation between data relating to the
-    provision of the EUDI Wallet and any other data held by the provider
+    provision of the EUDI Wallet and any other data held by the Provider
```
[the remark this answers is not in the document]
Comment (digeorgi) [reply docs/arf.md:2594]: Updated in accordance

File: docs/arf.md [hunk 11]
```diff
@@ -2745,7 +2745,7 @@ In parallel to the work described above, ENISA is requested to draft a
 dedicated European cybersecurity certification scheme for the EUDI
 Wallets under the CSA. Once available, this CSA scheme should replace
 the national transitory schemes mentioned above for the cybersecurity
-requirement it will cover. This scheme should be based on harmonised
+requirement it will cover. This scheme should be based on harmonized
```
[the remark this answers is not in the document]
Comment (digeorgi) [reply docs/arf.md:2748]: Updated in accordance

File: docs/arf.md [hunk 12]
```diff
@@ -2776,10 +2776,10 @@ Regulation, to facilitate certification.
 -   **Functional:** Describes functional requirements (i.e. what the
     EUDI Wallet should be able to do, such as provisioning of
     attributes). Most functional requirements are derived in the other
-    sections of the ARF coming from the Toolbox process.
+    Sections of the ARF coming from the Toolbox process.
```
[the remark this answers is not in the document]
Comment (digeorgi) [reply docs/arf.md:2779]: Updated in accordance

File: docs/arf.md [hunk 13]
```diff
@@ -2776,10 +2776,10 @@ Regulation, to facilitate certification.
...
 
 To provide recommendations on certification, the document describes 121
-requirements across five sections: legal requirements, scheme
+requirements across five Sections: legal requirements, scheme
```
[the remark this answers is not in the document]
Comment (digeorgi) [reply docs/arf.md:2782]: Updated in accordance

File: docs/arf.md [hunk 14]
```diff
@@ -2933,7 +2933,7 @@ suggestions, or concerns you may have regarding this document.
 To provide feedback on this document, please visit our GitHub
 repository. You can do so by navigating to the \"Issues\" tab and
 submitting a new issue or commenting on existing ones. Whether you\'ve
-spotted a typo, have a suggestion for clarifying a section, or want to
+spotted a typo, have a suggestion for clarifying a Section, or want to
```
[the remark this answers is not in the document]
Comment (digeorgi) [reply docs/arf.md:2936]: Updated in accordance

**Review by digeorgi — APPROVED** [review]
Checked for consistency in: 
1. Capitalization
2. en-GB spelling

**Comment by pinamiranda** [comment]
The proposed changes have apparently been addressed in other pull requests.

**Comment by skounis** [comment]
> The proposed changes have apparently been addressed in other pull requests.

@pinamiranda @digeorgi Verification is needed. I have noticed that the text in the `main` branch inconsistently capitalizes words (e.g., "section" versus "Section"), and there appears to be no active PR that addresses this issue.

**Pull request closed without merging**